CMakeLists.txt

```diff
@@ -13,3 +13,4 @@ add_subdirectory(ext)
 add_subdirectory(lib)
 add_subdirectory(subsys)
 add_subdirectory(drivers)
+add_subdirectory(tests/unity)
```

Comment:
I've seen people be lazy and do add_subdirectory(tests/junit) instead of re-organizing this into add_subdirectory(tests), so let's do the future-proof solution from the get go.

Re-write this to not add_subdirectory several levels down.